v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java

```diff
@@ -113,7 +112,7 @@ public interface Options extends PipelineOptions {
         order = 4,
         optional = true,
         description = "Cloud Storage directory for storing JSON files",
-        helpText = "The Cloud Storage path where the output JSON files can be stored.",
+        helpText = "The Cloud Storage path where the output JSON files are stored. For example: gs://your-bucket/your-path/.",
```

Comment:
The example will get auto-generated by the docs, so we don't need to add `For example: ...`